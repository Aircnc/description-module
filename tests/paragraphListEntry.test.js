import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import ParagraphListEntry from '../client/src/components/paragraphListEntry.jsx';

describe('ParagraphListEntry', () => {
  it('should render correctly', () => {
    var output = shallow(
      <ParagraphListEntry paragraph='mockParagraph'/>
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});