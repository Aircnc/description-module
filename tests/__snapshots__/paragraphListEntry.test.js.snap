// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ParagraphListEntry should render correctly 1`] = `
<styled.p>
  mockParagraph
</styled.p>
`;
